tripy/nvtripy/trace/ops/utils.py

```diff
@@ -70,3 +70,34 @@ def get_broadcast_in_dim(input_rank: int, output_rank: int) -> List[int]:
 
     assert len(broadcast_dimensions) == input_rank
     return broadcast_dimensions
+
+
+##
+## Datatype conversion
+##
+
+
+def get_trt_dtype_enum_str(dtype: "nvtripy.dtype") -> str:
```

Comment:
I think we should make this a property of `dtype` so we don't have to update multiple places when adding new dtypes.